Tasks/Gradle/CodeAnalysis/Common/BuildOutput.ts

```diff
@@ -59,14 +54,12 @@ export class BuildOutput {
     private getBuildDirectoryName(): string {
         switch (this.buildEngine) {
             case BuildEngine.Gradle:
-                {
-                    return 'build';
-                }
+                return 'build';
```

Comment:
preview also comes with a message advertising to the user what it brings and why they should "try it out".  If it's tests and linting then it doesn't offer anything - keep it 1.x and bump minor version.  If it's a refactoring or brings some new enticing capability that you want folks to try out, then mark as 2.x and take the opportunity for larger refactorings.  We shouldn't take a new major version lightly.  It has a substantial cost engineering wise (we publish tip of every major version every sprint to the DB) and it has a user cost - they have to make decisions whether to jump to 2.x (there's a drop down now).  Customers will also stick to 1.x until they explicitly jump to 2.x so you won't get usage of your new changes without them explicitly deciding to jump.